Jenkinsfile

```
@@ -1,20 +1,42 @@
-#!groovy
+#!/usr/bin/env groovy
+@Library('jenkins_global-libraries@main')_
 
-stage("Unit tests") {
-    node('') {
-        checkout scm
-
-        wrap([$class: 'AnsiColorBuildWrapper']) {
-            sh '''
-            SPARK_CONTAINER="spark-stat-analyser_run_$BUILD_NUMBER"
-
-            docker-compose -f docker-composer.test.yml build
-            docker-compose -f docker-composer.test.yml run --name $SPARK_CONTAINER spark-stat-analyser
-            docker cp $SPARK_CONTAINER:/srv/spark-stat-analyzer/junit.xml .
-            docker-compose -f docker-composer.test.yml down --remove-orphans
-            '''
-            junit 'junit.xml'
+pipeline {
+    agent any
+    options {
+        buildDiscarder(logRotator(numToKeepStr: '10'))
+    }
+    stages {
+        stage('fetch tag to be able to get them') {
```

```suggestion
        stage('fetch tags') {
```

is simpler, no? :smile: